.github/workflows/dev_data-tools_pr.yml

```diff
@@ -0,0 +1,62 @@
+name: Dev Data-Tools Pull Request
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - backend/models/**
+      - backend/data_tools/**
+      - backend/Dockerfile.data-tools
+
+env:
+  TERRAFORM_VERSION: "1.6.6"
```

Comment:
@devops-chris  Is there a better solution other than repeatedly specifying a terraform version variable in each yaml file? I see that we're doing that now in 5 existing files and this PR would make it 7--and that's just for the dev environment, based on the naming scheme if I'm reading it correctly. Is it desired to have to specify versions separately in variables in each file ?

Reply:
Resolved the TF versions in the PR #1885, but I am going to hold off on this PR until I know that data-tools can support automated deployment...